cubano-httpeasy/src/main/java/org/concordion/cubano/driver/http/HttpEasy.java

```diff
@@ -94,12 +96,12 @@
  * <b>Host and Certificate Verification</b>
  * </p>
  * <p>
- * There is no fine grained control, its more of an all or nothing approach:
+ * This will disable SSL and Hostname checking on HTTPS connections:
```

Comment:
Was this just for the next HttpEasy request?

Reply:
If so, can we have a test to lock this in. It will be important to offer some guarantees around this.

Reply:
It depends if you set the global default via `HttpEasy.withDefaults().trustAllEndPoints(true);` or per request via `HttpEasy.request().trustAllEndPoints(true);`

I've added some tests to illustrate this.